student-work/joe-wainer/api-final-project/error-codes.md

```diff
@@ -0,0 +1,13 @@
+# Error Codes
+
+These are the HTTP status codes for a successful or failed API call. 
+
+| Status | Code Description | Customer Facing Description  |
+| --- | --- | --- |
+| 200 - OK  | Your request was successful. | Your order will be ready soon.  |
+| 400 - Bad Request | Client error. You may be missing a required field.  | Oops something went wrong. Please recheck your order for any missing info or toppings.   |
+| 401 - Unauthorized Request | Access denied. Check your API token and access privileges.  | Error. Did you verify your mobile number? If not, please verify your mobile number, then order again.  |
+| 402 - Request Failed | Your request timed out.  | Sorry, you took too long. Please place your order again.  |
+| 404 - Not Found  | The server could not find what you requested. | Something went wrong. Try again or ask a member of staff for assistance  |
```

Comment:
```suggestion
| 404 - Page Not Found  | The server could not find the page you requested. | This can happen if there is a spelling error or if the page doesn't exist. Check your spelling and if it is correct, ask a member of staff for assistance.  |
```